discord/user.py

```diff
@@ -128,7 +131,7 @@ def avatar_url_as(self, *, format, size=1024):
             if self.avatar.startswith('a_'):
                 format = 'gif'
             else:
-                format = 'webp'
+                format = static_format
```

Comment:
png,webp,gif,jpeg are the only supported static formats. everything else will throw a 415 status code. you could save the caller a lot of trouble by validating this and perhaps throwing an exception.